deploy/helm/charts/Chart.yaml

```diff
@@ -16,5 +16,5 @@ sources:
 - https://github.com/openebs/zfs-localpv
 dependencies:
   - name: crds
-    version: 2.6.0
+    version: 2.7.0-develop
```

Comment:
Do we still need to keep bumping this one btw?

Reply:
I am not sure what to do here. I just bumped as that's what we have been doing.

Reply:
Yeah that's fine, it's more of a general question. CC @niladrih